packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -12,20 +12,25 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
     opacity: 1
   };
 
+  const inputIconAlt = palette.neutralSecondary;
+  const inputIconAltHovered = palette.neutralPrimary;
+  const inputBorderDisabled = palette.neutralLighter;
+  const inputBackgroundHovered = palette.neutralLighter;
+  const inputBackgroundDisabled = palette.neutralLighter;
+
   return {
     root: [
       'ms-SearchBox',
       fonts.medium,
       normalize,
       {
-        color: palette.neutralPrimary,
+        color: semanticColors.inputText,
         backgroundColor: semanticColors.inputBackground,
         display: 'flex',
         flexDirection: 'row',
         flexWrap: 'nowrap',
         alignItems: 'stretch',
-        // The 1px top and bottom padding ensure the input field does not overlap the border
-        padding: '1px 0 1px 4px',
```

Comment:
Is this not needed anymore?

Reply:
Per our Teams convo I have placed it back